src/shared/logs-show.c

```diff
@@ -1227,14 +1205,60 @@ static int output_cat(
         return 0;
 }
 
+static int get_dual_timestamp(sd_journal *j, dual_timestamp *ret_ts, sd_id128_t *ret_boot_id) {
```

Comment:
Why not to implement tiny wrapper `journal_get_dual_timestamp()` in sd-journal.c ?

Reply:
`ret_boot_id` is not used??

Reply:
> Why not to implement tiny wrapper journal_get_dual_timestamp() in sd-journal.c ?

I would agree with @poettering and keep it like this. :)

> ret_boot_id is not used??

Line `1293`:

```c
r = sd_journal_get_monotonic_usec(j, &ret_ts->monotonic, ret_boot_id);
```